### ISSUE TITLE
Calibrate Robot to Pipette

### ISSUE THREAD

**Issue by umbhau** [Opentrons/opentrons#699]
As a robot user, I would like my robot to automatically calibrate to whichever pipette I am using.

## Acceptance Criteria
- Store y-calibration (and tbd other calibration) on robot
- Automatically set calibration when pipette is attached.

## Resources
- Default calibration values:
https://docs.google.com/spreadsheets/d/1KOGUng337g-blA8Kw40QwSXYjWRodoclLBR8LQOQYd0/edit#gid=0

## Implementation Notes
- We will eventually move pipette calibration information to pipette memory
- Robots will have to check both pipette (first priority) and internal config file (second priority) for relevant calibration data.

**Comment by andySigler** [comment]
Explicitly instantiated pipettes by their model name could look like:
```python
p10 = instruments.P10_Single(mount='left')
m300 = instruments.P300_Multi(mount='right')
```

**Comment by andySigler** [comment]
Also fixes #561
